sycl/include/CL/sycl/detail/pi.h

```diff
@@ -278,6 +281,8 @@ typedef enum {
       CL_DEVICE_CROSS_DEVICE_SHARED_MEM_CAPABILITIES_INTEL,
   PI_DEVICE_INFO_USM_SYSTEM_SHARED_SUPPORT =
       CL_DEVICE_SHARED_SYSTEM_MEM_CAPABILITIES_INTEL,
+  // Intel UUID extension.
+  PI_DEVICE_INFO_UUID = CL_DEVICE_UUID_KHR,
```

Comment:
Is `CL_DEVICE_UUID_KHR` not supported by OpenCL RT, but is in the header?

Reply:
yes, that is the case at this point.

Reply:
OK